lib/views/library_card.js

```diff
@@ -67,5 +81,13 @@ export class LibraryCard extends View {
 		this.find('#view-on-github').on('click', () => {
 			shell.openExternal(library.repository);
 		});
+
+		this.find('#add-to-project').on('click', () => {
+			const command = new LibraryAddCommand();
+			const apiClient = fetchApiClient();
+			const site = new DevLibraryAddSite(apiClient, core().getProjectDir());
+			site.setLibraryIdent(library.name);
```

Comment:
probably a good idea to also add the version here in case multiple versions are ever displayed (can we really assume it's always the latest version? e.g. when viewing both my own public and private libraries.)